app/controllers/stories_controller.rb

```diff
@@ -200,7 +200,7 @@ def handle_organization_index
       .limited_column_select
       .order(published_at: :desc).page(@page).per(8))
     @organization_article_index = true
-    set_organization_json_ld
+    @organization_json_ld = Articles::JsonCreator.organization(@organization)
```

Comment:
Hi @brunnohenrique, I'm not convinced this is the right solution. This is not about creating JSON per se but it's just a different representation of the org object, or user object or article object. I think we should respect that and use serializers, similar to `app/serializers`

In this case you're serializing `JSON-LD`, so I would imagine something like `app/serializers/json_ld/article_serializer.rb` with the same structure of the existing ones. Same for the other models

cc @citizen428

Reply:
Yes, I recently mentioned this in another PR too (can't remember which one though): if the main purpose of a service is generating a JSON response, it's a serializer in disugise. 😉